pom.xml

```diff
@@ -858,7 +859,13 @@
       <dependency>
         <groupId>org.testcontainers</groupId>
         <artifactId>testcontainers</artifactId>
-        <version>1.14.3</version>
+        <version>${testcontainers.version}</version>
```

Comment:
just double checking that previously it was 1.14.3 and now it is upgraded to 1.18.3

Reply:
yeah so I can confirm the only places using testcontainer lib is from this line:

```
dora/tests/src/test/java/alluxio/server/worker/WorkerMetadataSyncIntegrationTest.java:28:
import org.testcontainers.shaded.com.google.common.collect.ImmutableMap;
```
which is totally unnecessary and could be independent of which version of testcontainer  we have originally